project3.0/css/style.css

```diff
@@ -1,3 +1,9 @@
+:root {
+    --font-main: 'Heebo' , sans-serif;
+    --font-style: var(--font-style);
```

Comment:
--font-main correct,

--font-style / --font-weight wrong.

Need to be like this:
--font-style: italic;
--font-weight: 700

Reply:
But better use CSS vars for font-family, colors, some same margin's/padding's